x/auth/vesting/keeper/genesis.go

```diff
@@ -0,0 +1,19 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/vesting/exported"
+)
+
+// InitGenesis - Init vesting store state
+func (vk VestingKeeper) InitGenesis(ctx sdk.Context) {
+	var vestingAccounts []exported.VestingAccount
+	vk.accountKeeper.IterateAccounts(ctx, func(account authtypes.AccountI) bool {
+		if va, ok := account.(exported.VestingAccount); ok {
+			vestingAccounts = append(vestingAccounts, va)
+			vk.AddVestingAccount(ctx, va.GetAddress())
```

Comment:
Dose it only track vesting accounts on genesis, not the ones created at runtime?